frontend/src/components/people/person-card.tsx

```diff
@@ -1,10 +1,37 @@
 import React, { FC } from "react";
 import { IPersonProps } from '../../interfaces/person.dto';
-import { Card } from '@material-ui/core';
+import { Avatar, Card } from '@material-ui/core';
 
-export const PersonCard: FC<IPersonProps> = (props: IPersonProps) => {
+import "./person-card.css"
+
+export const PersonCard: FC<IPersonProps> = ({ 
+  email, profile_picture, title, company, first_name, last_name 

```

Comment:
Can't we simplify this by having `personProps: IPersonProps` instead of `{ email, profile_picture, title, company, first_name, last_name }: IPersonProps`?

Reply:
This can be tricky here, but I think it's much easier to implement later. I prefer this approach instead of doing: personProps.email, personProps.first_name, etc...

Reply:
I think we can benefit from TypeScript a little bit here, actually. Let's say that we modify the IPersonProps interface in the future, and add a bunch of new props. We would then need to manually add the object destructuring in the PersonCard's component for each of the new props.

Let's say we then extend the IPersonProps interface for something more specific, such as IStundentProps, ITeacherProps and IAdvisorProps. As we extended the IPersonProps interface and added new props, we would need then to add this destructuring in every single component that references an IPersonProps prop that changed. I would go with the object declaration instead of destructuring the props, is more safe. I do understand that for a single component on a simple application it doesn't really matter, but I think is better to be using objects and referencing their properties now with TypeScript.